### DISCUSSION TITLE
ERROR:binance.websocket.binance_client_factory

### DISCUSSION THREAD

**Discussion by andre9966** [escomputers/BASD#44] [General]
I run main from last alpha release, i set correctly configuration parameters but I have this error
ERROR:binance.websocket.binance_client_factory:Lost connection to Server. Reason: [Failure instance: Traceback (failure with no frames): <class 'twisted.internet.error.ConnectionLost'>: Connection to the other side was lost in a non-clean fashion: Connection lost.
I have installed in my pc Python 3.11

**Comment by escomputers** [comment]
Please post comments [here](https://github.com/escomputers/BASD/issues/45)
